src/librustdoc/html/render/write_shared.rs

```diff
@@ -791,10 +797,12 @@ impl TraitAliasPart {
     }
 }
 
+#[derive(PartialEq, Eq)]
```

Comment:
logic error: `Eq` and `Ord` impls disagree on the definition of equality, as `Ord` only checks the name.

this can easily be fixed by writing a `PartialEq` impl that defers to `Ord`:

```rust
impl PartialEq for Implementor {
    fn eq(&self, other: &Self) -> bool {
        self.cmp(other) == Ordering::Equal
    }
}
```

Reply:
Wait, in which would it be an issue to have this difference? Ordering and equality don't have to be equivalent.